torch_xla/csrc/ops/dynamic_ir.cpp

```diff
@@ -60,7 +60,7 @@ XlaOpVector SizeNode::Lower(LoweringContext* loctx) const {
   return ReturnOp(xla::GetDimensionSize(input, this->dim_), loctx);
 }
 
-std::string SizeNode::ToString() const { return "SizeNode"; }
+std::string SizeNode::ToString() const { return "aten::size"; }
```

Comment:
Update the `xla::ShapeUtil::MakeShape(xla::S64,` to `GetShapeDimensionType`

Reply:
What do you mean? "SizeNode" is already using your fix:
```
SizeNode::SizeNode(torch::lazy::Value input, size_t dim)
    : XlaNode(torch::lazy::OpKind{c10::Symbol::fromQualString("aten::size")},
              {input},
              xla::ShapeUtil::MakeShape(
                  GetShapeDimensionType(/*device=*/nullptr), {}),
```
For other nodes, I plan to update in follow-up PRs.

Reply:
If test can pass I am ok, but I think one of the cpu test failed because SizeAdd hardcode the s64 as type.

Reply:
this is fixed.